modules/linear_boltzmann_solvers/a_lbs_solver/lbs_solver.h

```diff
@@ -457,23 +457,30 @@ class LBSSolver : public opensn::Solver
    */
   virtual void SetPrimarySTLvectorFromGroupScopedPETScVec(int first_group_id,
                                                           int last_group_id,
-                                                          Vec x_src,
+                                                          Vec x,
                                                           std::vector<double>& y);
 
   /**
    * Assembles a PETSc vector from multiple groupsets.
    */
-  virtual void SetMultiGSPETScVecFromPrimarySTLvector(const std::vector<int>& gs_ids,
+  virtual void SetMultiGSPETScVecFromPrimarySTLvector(const std::vector<int>& groupset_ids,
                                                       Vec x,
                                                       PhiSTLOption which_phi);
 
   /**
    * Disassembles a multiple Groupset PETSc vector STL vectors.
    */
-  virtual void SetPrimarySTLvectorFromMultiGSPETScVecFrom(const std::vector<int>& gs_ids,
-                                                          Vec x_src,
+  virtual void SetPrimarySTLvectorFromMultiGSPETScVecFrom(const std::vector<int>& groupset_ids,
+                                                          Vec x,
                                                           PhiSTLOption which_phi);
 
+  /**
+   * A method for post-processing an adjoint solution.
+   *
+   * @note This does nothing for diffusion-based solvers.
+   */
+  virtual void ReorientAdjointSolution(){};
```

Comment:
There is a doc-string for the overridden method which details what happens. In documentation, if you look at `DiscreteOrdinatesdSolver::ReorientAdjointSolution` what you are saying would be there. I did not put it here since this routine in `LBSSolver` does not do that. I can still add it here if you'd like.